src/Raven.Client/Documents/Indexes/AbstractIndexCreationTask.cs

```diff
@@ -166,6 +165,25 @@ protected object CreateField(string name, object value)
             throw new NotSupportedException("This can only be run on the server side");
         }
 
+#if FEATURE_DATEONLY_TIMEONLY_SUPPORT
+        /// <summary>
+        /// Allow to index DateOnly as Ticks
+        /// </summary>
+        protected DateOnly AsDateOnly(object field)
```

Comment:
shouldn't this be in AbstractCommonApiForIndexes ?

Reply:
Moved